dist/javascript.js

```diff
@@ -3,94 +3,266 @@
 
 // Please, refer to instructions for adding a custom metric in almanac.js.
 
+function fetchWithTimeout(url) {
+    var controller = new AbortController();
+    setTimeout(() => {controller.abort()}, 5000);
+    return fetch(url, {signal: controller.signal});
+  }
+
 const requests = $WPT_REQUESTS;
 
-return JSON.stringify({
+let getSourceMaps = (async () => {
+  const $$ = (s) => [...document.querySelectorAll(s)];
+  const sourcemapRegex = /\/\/[#@] sourceMappingURL=(.+?)$/;
```

Comment:
Do you also want to capture the [`SourceMap`](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/SourceMap) header?

Reply:
Is there a custom-metric already looking at the SourceMap header? From last year's chapter, usage is low with 0.1% of pages using this.

https://almanac.httparchive.org/en/2021/javascript#source-maps

Reply:
Here's an example:

https://github.com/HTTPArchive/custom-metrics/blob/main/dist/responsive_images.js#L458

I agree usage is probably too low to matter, so I'll leave it up to you.

Reply:
Yeah, will leave out for now.